index.js

```diff
@@ -94,13 +95,14 @@ function Argv (processArgs, cwd) {
         return self;
     };
 
-    self.default = function (key, value) {
+    self.default = function (key, value, defaultDescription) {
         if (typeof key === 'object') {
             Object.keys(key).forEach(function (k) {
                 self.default(k, key[k]);
```

Comment:
with the concept of defaultDescription, one thing I don't like is that you can't set a defaultDescription if you pass in an object for `key` ... I continue to be so so about the concept of a `defaultDescription`, is wrapping the method in a named function too weird?